ex1-files/style.css

```diff
@@ -0,0 +1,232 @@
+body {
+  margin: 0%;
+}
+
+nav {
+  height: 46px;
+}
+
+nav .logo {
+  float: left;
+  padding-top: 16px;
+  padding-left: 36px;
+  box-sizing: border-box;
+}
+.logo a {
+  text-decoration: none;
+}
+.header-actions {
+  padding-top: 19px;
+  width: 60%;
+  float: right;
+  text-align: right;
+  box-sizing: border-box;
+}
+
+.header-actions a {
+  padding-left: 25px;
+  padding-right: 25px;
+  text-decoration: none;
+}
+
+.header-actions a:hover {
+  text-decoration: underline;
+  border-bottom: solid 4px black;
+  text-align: center;
+}
+
+.items a {
+  font-size: 14px;
+  color: rgb(0,0,0);
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+}
+
+.blueprint {
+  background-image: url("../ex1-files/sample1.jpg");
+  background-size: cover;
+  background-position: bottom center;
```

Comment:
you can combine the background image and background position:
`background: url("../ex1-files/sample1.jpg") bottom center;`

Reply:
yeah, thanks.